project/Build.scala

```diff
@@ -73,7 +73,9 @@ object DottyBuild extends Build {
       javaSource in Test := baseDirectory.value / "test",
       resourceDirectory in Compile := baseDirectory.value / "resources",
       unmanagedSourceDirectories in Compile := Seq((scalaSource in Compile).value),
+      unmanagedSourceDirectories in Compile += baseDirectory.value / "dottydoc" / "src",
```

Comment:
Yes, but the problem is getting it on to the classpath in sbt when compiling the bridge. If it is part of dotty - that of course does not become a problem. Again, even if that was fixed by making a PR to sbt, they would need to include both the dottydoc jar as well as the dottydoc-client jar.